micro-ui/web/micro-ui-internals/packages/css/CHANGELOG.md

```
@@ -0,0 +1,141 @@
+# Changelog
+
+## [1.8.2-beta.49]
+- Updated Loader with text styles
+
+## [1.8.2-beta.28]
+- Fixed the loader style 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)